elasticpress-woocommerce.php

```diff
@@ -296,13 +309,19 @@ function epwc_translate_args( $query ) {
 						$query->set( 'order', 'desc' );
 						break;
 					default:
-						$query->set( 'orderby', 'menu_order title date meta._price.long' ); // Order by menu and title.
+						$query->set( 'orderby', 'menu_order title date' ); // Order by menu and title.
 						$query->set( 'order', 'asc' );
 				}
 			} else {
-				$query->set( 'orderby', 'menu_order title date meta._price.long' ); // Order by menu and title.
-				$query->set( 'order', 'asc' );
+					$query->set( 'orderby', 'menu_order title date' ); // Order by menu and title.
+					$query->set( 'order', 'asc' );
 			}
+		} // Conditional check for orders
+		elseif ( in_array( $post_type, array( 'shop_order', 'shop_order_refund' ) ) || ( is_array( $post_type ) && ! array_diff( $post_type, $supported_post_types ) ) ) {
```

Comment:
The second part of the conditional is confusing. Perhaps it should be it's own conditional?